### DISCUSSION TITLE
是否可以优化启动器启动速度以及页面打开速度性能。

### DISCUSSION THREAD

**Discussion by cuicanx** [Meloong-Git/PCL#4885] [提问] [answered]
**是否可以优化启动器启动速度以及页面打开速度性能。**

**Comment by Pigeon0v0** [comment]
你还想怎么优化

**Reply by MoYuan-CN** [reply]
详细描述你希望优化的内容……

**Comment by SALTWOOD** [comment, marked as answer]
这基本已经是极限了，刨除 WPF 启动，还要加载 Newtonsoft.Json NAudio 什么的一堆依赖，还要读注册表加载主题什么的，多等几秒也不会咋样

**Comment by wuliaodexiaoluo** [comment]
>是否可以优化启动器启动速度以及页面打开速度性能。

这边建议找 （骚扰） 下 Microsoft
